.github/ISSUE_TEMPLATE/p_doc.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Documentation 📘
+about: Suggest a documentation item for Eclipse Che
+title: ''
+labels: 'kind/docs'
+assignees: ''
```

Comment:
@rkratky we can make you as default assignee as well there

Also we can add the template that you would want for documentation items.